packages/itwin/reports-config-widget/src/widget/components/AddMappingsModal.tsx

```diff
@@ -5,7 +5,8 @@
 import { Modal, Table, tableFilters } from "@itwin/itwinui-react";
 import React, { useEffect, useMemo, useState } from "react";
 import type { Mapping } from "@itwin/insights-client";
-import { REPORTING_BASE_PATH, ReportingClient } from "@itwin/insights-client";
+import { ReportsClient } from "@itwin/insights-client";
+import { MappingsClient, REPORTING_BASE_PATH } from "@itwin/insights-client";
```

Comment:
Merge these as well.

Reply:
Merged, also double checked all insights-client imports and there should be no more duplicates.